webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -171,37 +172,50 @@ export class WebhookDisplayTable extends Component {
           <Redirect to={this.props.match.url + "/create"} />
         )
       } else {
+        const { namespace } = this.props;
         // There are webhooks so display table
         const headers = [
           {
-            key: 'name',
-            header: 'Name',
+            key: "name",
```

Comment:
Why the change to double quotes here? Consistent quote style should be enforced by the linter.